src/member/member.module.ts

```diff
@@ -1,9 +1,12 @@
 import { Module } from '@nestjs/common';
 import { MembersController } from './member.controller';
 import { MembersService } from './member.service';
+import { MemberRepository } from './member.repository';
+import { PrismaService } from '../prisma/prisma.service';
 
 @Module({
   controllers: [MembersController],
-  providers: [MembersService],
+  providers: [MembersService, MemberRepository, PrismaService],
```

Comment:
```suggestion
  providers: [MembersService, MemberRepository],
```
프리즈마는 글로벌로 등록돼있지 않나요?